system/system_monitor/include/hdd_reader/hdd_reader.hpp

```diff
@@ -88,7 +98,11 @@ struct HDDInfo
     ar & temp_;
     ar & power_on_hours_;
     ar & total_data_written_;
+    ar & recovered_error_;
+    ar & is_valid_temp_;
+    ar & is_valid_power_on_hours_;
     ar & is_valid_total_data_written_;
+    ar & is_valid_recovered_error_;
```

Comment:
@v-nakayama7440-esol
Sharing request for S.M.A.R.T. information and request to lazy unmount in one struct would not be good practice because there would be a risk for this struct to be bigger and bigger for further improvement and this struct would be hard to be managed. Maybe significant changes, but these functionalities should be separated into message ids based on usage.

Reply:
@ito-san 
I separated requests for S.M.A.R.T. and lazy unmounts.
c97b8bc4436976d2d76d0efc701bf5ba54404eec

Reply:
Thanks.